book/src/ledger-replication.md

```diff
@@ -30,12 +29,12 @@ core. The total space required for verification is `1_ledger_segment +
 
 Validators for PoRep are the same validators that are verifying transactions.
 They have some stake that they have put up as collateral that ensures that
-their work is honest. If you can prove that a validator verified a fake PoRep,
-then the validator will not receive a reward for that storage epoch.
+their work is honest. If a replicator can prove that a validator verified a fake
```

Comment:
validators don't have stake at risk anymore, stakers of the validator's vote account do...

Reply:
I've removed this from the book. For now I'm not considering slashing, just denying rewards.